### PR TITLE
Issue 215 fix build issue

### PR DESCRIPTION
**Pull request by urb4nc4rl**
Pin arduino-esp32 package to version 1.0.5 to fix [issue 215](https://github.com/doudar/SmartSpin2k/issues/215)

### CODE REVIEW

**Comment by kadaan** [comment]
Can we use https://github.com/espressif/arduino-esp32/releases/tag/1.0.6

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Pin arduino-esp32 package to version 1.0.5 to fix build issue
```
Comment (kadaan) [inline CHANGELOG.md:9]: Ahh. You are using 1.0.6, but this changelog entry is wrong.

**Review by kadaan — CHANGES_REQUESTED** [review]

**Comment by kadaan** [comment]
Related to https://github.com/doudar/SmartSpin2k/discussions/201#discussion-3339992

**Comment by kadaan** [comment]
Realistically, this will need at least a cursory ride test to be sure that changing from master to 1.0.6 didn't break anything.

File: CHANGELOG.md (hunk 1 above)
Comment (urb4nc4rl) [reply CHANGELOG.md]: Apologies for typo, should be fixed up now

Comment (doudar) [reply CHANGELOG.md]: Thanks for the PR guys. 

I'm away from my main computer (I'm checking in from a quick break from class), can anyone confirm what version we have been building with? If anyone can confirm it's 1.06, I'm good with the merge. No really need to keep develop branch absolutely rock stable as I will build auto update binaries from master.

Comment (kadaan) [reply CHANGELOG.md]: @doudar We had been building with master, so who knows...

**Review by doudar — APPROVED** [review]

**Review by kadaan — APPROVED** [review]
Pending a ride test.

**Comment by doudar** [comment]
Will be merged as soon as anyone can post that this successfully built and a short ride worked. 

(I'll take your word for it) :)

**Comment by doudar** [comment]
> Pending a ride test.

Compiles and runs good on my laptop. tested with btsimulator.html

**Pull request merged**